app/models.py

```diff
@@ -344,13 +363,32 @@ def is_public(self):
     @property
     def is_agency(self):
         """
-        Checks to see if the current user is an agency user
+        Check to see if the current user is an agency user.
 
         AGENCY_USER = 'Saml2In:NYC Employees'
 
         :return: Boolean
         """
-        return self.auth_user_type in user_type_auth.AGENCY_USER_TYPES and self.agency_ein is not None
+        return self.auth_user_type in user_type_auth.AGENCY_USER_TYPES and self.agencies is not None
+
+    @property
+    def default_agency_ein(self):
+        """
+        Return the Users default agency ein.
+        :return: String
+        """
+        agency = AgencyUsers.query.join(Users).filter(AgencyUsers.is_primary_agency == True).one_or_none()
```

Comment:
This query raises MultipleResultsFound error. 64 results are being returned from this query. If this query is intended for returning the default agency ein for a single user, should it take in a guid as an argument and query with it as well?